### PR TITLE
Add low poly hands with arms

### PR DESCRIPTION
**Pull request by JuliaRonnebergerUL**
## Pull Request Templates

Switch template by going to preview and clicking the link - note it not work if you've made any changes to the description.

- [default.md](?expand=1) - for contributions to stable packages.
- [lightweight.md](?expand=1&template=lightweight.md) - for contributions to pre-release/preview packages use the lightweight merge request template. The quality threshold for reviewing is lower - it prioritizes a lean review process.
- [release.md](?expand=1&template=release.md) - for release merge requests.

**You are currently using: default.md**

Note: these links work by overwriting query parameters of the current url. If the current url contains any you may want to amend the url with `&template=name.md` instead of using the link. See [query parameter docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request) for more information.

## Summary

Adding low poly hands with arms and adding them to the standard rigged hands example scene

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [x] Pair review with a member of the QA team.
- [x] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [x] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [x] Code reviewed.
- [x] Non-code assets e.g. Unity assets/scenes reviewed.
- [ ] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [ ] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

_If this MR closes any JIRA issues list them below in the form `Closes PROJECT-#`_

### CODE REVIEW

**Comment by Dwarph** [comment]
image
Lack of a face on the end of the arm feels wrong?

**Comment by Dwarph** [comment]
mesh scaling doesn't seem to be working for them - my finger tips are quite far apart when I touch them together

https://user-images.githubusercontent.com/17143693/179464181-7e2bee10-2f11-4103-9331-d20bf0086ef2.mp4

**Comment by Dwarph** [comment]
The material assigned to the mesh is a different one to Low Poly Hands

**Comment by Dwarph** [comment]
URP & HDRP scenes should be fine with the old low poly materials :)

File: Packages/Tracking/Hands/Runtime/Scripts/ShowArm.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
+using System.Collections;
```
Comment (MattGrayUL) [inline Packages/Tracking/Hands/Runtime/Scripts/ShowArm.cs:1]: Copyright header? :)

**Comment by JuliaRonnebergerUL** [comment]
Testing that I have done:
1) new hands work
- elbow follows elbow tracking data
- the hand model looks right from all sides and in different poses
2) hands are part of the example scenes and work there
- all three rigged hands example scenes include the new hands
- hands look good in URP and HDRP
3) hand scaling works
- the standard tests we do for hand scaling pass (eg. changing the scale offset, changing fingertip offsets, enabling / disabling scale)

@Greg-UL  Is there anything else that you think needs to be tested before this is merged?

**Review by MattGrayUL — APPROVED** [review]

**Comment by Greg-UL** [comment]
> Testing that I have done:
> 
> 1. new hands work
> 
> * elbow follows elbow tracking data
> * the hand model looks right from all sides and in different poses
> 
> 2. hands are part of the example scenes and work there
> 
> * all three rigged hands example scenes include the new hands
> * hands look good in URP and HDRP
> 
> 3. hand scaling works
> 
> * the standard tests we do for hand scaling pass (eg. changing the scale offset, changing fingertip offsets, enabling / disabling scale)
> 
> @Greg-UL Is there anything else that you think needs to be tested before this is merged?

Is this working alright across Unity versions too?
Can we activate deactivate the elbow tracking whilst using the arm models?
What's the interaction like for switching showArm like? Pre editor preview mode switch on/off, during editor preview mode on/off.
How does the model react to losing elbow tracking?
Model in different positions - Elbows close to the face, outstretched arms, arms close to body.

**Comment by JuliaRonnebergerUL** [comment]
@Greg-UL 
> Is this working alright across Unity versions too? 

tested on Unity 2020 and Unity 2021
> Can we activate deactivate the elbow tracking whilst using the arm models? 

You can toggle `showArm` while using the model, and the arm will disappear / reappear
> What's the interaction like for switching showArm like? Pre editor preview mode switch on/off, during editor preview mode on/off. 

Video here: https://ultrahaptics.box.com/s/okldmyjl9k633sjx5aumqtgzhdeav5rw
> How does the model react to losing elbow tracking? 
 
I don't think it's possible to lose elbow tracking? The elbow will be tracked if the hand is tracked. And if the hand isn't tracked anymore, the HandEnableDisable script will disable the hand model
> Model in different positions - Elbows close to the face, outstretched arms, arms close to body.

Can also be seen on video here: https://ultrahaptics.box.com/s/okldmyjl9k633sjx5aumqtgzhdeav5rw

**Comment by Greg-UL** [comment]
> @Greg-UL
> 
> > Is this working alright across Unity versions too?
> 
> tested on Unity 2020 and Unity 2021
> 
> > Can we activate deactivate the elbow tracking whilst using the arm models?
> 
> You can toggle `showArm` while using the model, and the arm will disappear / reappear
> 
> > What's the interaction like for switching showArm like? Pre editor preview mode switch on/off, during editor preview mode on/off.
> 
> Video here: https://ultrahaptics.box.com/s/okldmyjl9k633sjx5aumqtgzhdeav5rw
> 
> > How does the model react to losing elbow tracking?
> 
> I don't think it's possible to lose elbow tracking? The elbow will be tracked if the hand is tracked. And if the hand isn't tracked anymore, the HandEnableDisable script will disable the hand model
> 
> > Model in different positions - Elbows close to the face, outstretched arms, arms close to body.
> 
> Can also be seen on video here: https://ultrahaptics.box.com/s/okldmyjl9k633sjx5aumqtgzhdeav5rw

Thanks Julia - that's brilliant.
All good from my perspective.

**Pull request merged**